### ISSUE TITLE
Changing date does not change model

### ISSUE THREAD

**Issue by zhikin2207** [nkalinov/ng2-datetime#214]
v. 1.4.0

Step to reproduce:
1. Open demo example (https://nkalinov.github.io/ng2-datetime/)
2. Scroll to the first example with options
3. Manually change date from 10 to 11 in input (remove 0 and add 1)
4. Click somewhere to hide date selection popup

![gif](https://user-images.githubusercontent.com/4587659/36787137-b14ab408-1c91-11e8-80cd-acb3b930ff24.gif)

**Comment by htrex** [comment]
Yes, this is a bug and there's a pending PR #213.
Please @nkalinov can you merge it and release a new version?

**Comment by htrex** [comment]
No, after some tests and code checking seems PR #213 is for another issue.
The problem seems to be related to the bootstrap-datepicker lib, have a look at the issue https://github.com/uxsolutions/bootstrap-datepicker/issues/2325 and problematic commit https://github.com/uxsolutions/bootstrap-datepicker/commit/110b12072cf424480a7fe0cd10ec917458a2e4a8

removing ``&& fromArgs`` from the bootstrap-datepicker lib solved my issue
